### PR TITLE
fixes 0 samples returned on certain 1HZ data requests (issue 59)

### PR DESCRIPTION
**Pull request by derick-hess**

### CODE REVIEW

**Comment by derick-hess** [comment]
hmm I think It changed the whole file again because I committed from my windows machine.

**Comment by nick-iris** [comment]
Yeah it is something with the new line characters being different on Windows. I'll reset this commit and then push the change from my machine so that we have a record of what changed.

**Comment by derick-hess** [comment]
Thanks, yeah I should just push changes form my virtual machine or wait until I get into work next time

File: kitchen/ph5API.py [hunk 1]
```diff
@@ -1486,210 +1486,242 @@ def sort_on_epoch (a, b) :
...
+            stringtime = co.start_time.getFdsnTime ()
+            filename = 'DATA/{0}_{1}_{2}.sac'.format (das, stringtime, co.das_t[0]['channel_number_i'])
+            #traces.append (t)
+            print co.start_time.epoch (fepoch=True), stringtime, t.stats.starttime, t.stats.endtime
+            t.write (filename, 'SAC')
+            
+        t1 = t2 + (1./float (sr))
+        t2 = t1 + 3600.        
+            
+    print
```
Comment (nick-iris) [inline kitchen/ph5API.py]: This print statement should be removed.

Comment (nick-iris) [inline kitchen/ph5API.py]: Actually never mind. It is in the __main__ method  so shouldn't matter.

**Pull request merged**